drivers/ec_host_cmd_periph/ec_host_cmd_periph_espi.c

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_ec_host_cmd_periph_espi
+
+#include <string.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/ec_host_cmd_periph/ec_host_cmd_periph.h>
+#include <zephyr/drivers/espi.h>
+#include <zephyr/mgmt/ec_host_cmd.h>
+
+BUILD_ASSERT(DT_HAS_COMPAT_STATUS_OKAY(DT_DRV_COMPAT), "No eSPI peripheral node");
```

Comment:
hey you can make `EC_HOST_CMD_PERIPH_ESPI` depend on `DT_HAS_ZEPHYR_EC_HOST_CMD_PERIPH_ESPI` and then this build assert should become redundant.

Reply:
Nice idea, thanks